faqs/galaxy/datatype_tabular.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Indentifying and Formatting Tabular Dataset 
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+---
+ 
+
+- A Tabular datatype is human readable and has tabs separating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+1. Common tabular datatypes are `.bed`, `.gtf`, `.interval`, or `.txt`.
+2. The datatype metadata attribute can be directly reassigned to tabular format data.
+3. Click the {% icon galaxy-pencil %} ✏️ icon to reach the **_Edit Attributes_** form. 
```

Comment:
We try to always also include the text description of the icon, followed by the icon itself, for accessibility reasons

```suggestion
3. Click the pencil icon {% icon galaxy-pencil %} to reach the **_Edit Attributes_** form. 
```